jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/EnvUtilTest.java

```diff
@@ -397,21 +399,25 @@ public void testLoadTimestampShouldLoadFromFile() throws Exception {
     @Test
     public  void testIsWindowsFalse(){
         //Given
-        new SystemMock().put("os.name", "random");
+        System.setProperty("os.name", "random");
+        // TODO : Replace this when https://github.com/eclipse/jkube/issues/958 gets fixed
         //When
         boolean result= EnvUtil.isWindows();
         //Then
         assertThat(result).isFalse();
+        System.clearProperty("os.name");
     }
 
     @Test
     public  void testIsWindows(){
         //Given
-        new SystemMock().put("os.name", "windows");
+        System.setProperty("os.name", "windows");
+        // TODO : Replace this when https://github.com/eclipse/jkube/issues/958 gets fixed
         //When
         boolean result= EnvUtil.isWindows();
         //Then
         assertThat(result).isTrue();
+        System.clearProperty("os.name");
```

Comment:
This is an already existing system property. We need to be careful while clearing it out. Let's change it like this:

```java
    @Test
    public  void testIsWindows(){
        String oldOsName = System.getProperty("os.name");
        System.setProperty("os.name", "windows");

        //Given


        //When

        //Then

        System.setProperty("os.name", oldOsName);
    }
```